week-3/20-labels/index.html

```diff
@@ -12,8 +12,9 @@
     <div class="site-wrapper">
       <main>
         <form method="GET" action="https://www.google.com/search">
-          <!-- Add a label here -->
-          <input type="text" name="q" />
+          <laple for="search-input"> Search on Google</laple>
```

Comment:
There's a typo here. It's a good idea to review your code changes using `git diff` before committing them.

Reply:
thanks for comment i got it and changed it